CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 This document outlines major changes between releases.
 
+## 0.18.0 (16 Dec 2021)
```

Comment:
Use format from keepachangelog.com. Then Sami-chan will be able to parse it for auto release.
```suggestion
## [0.18.0]- 2021-12-16
```